### DISCUSSION TITLE
Deploy to Azure Container

### DISCUSSION THREAD

**Discussion by rfenyves** [docusealco/docuseal#245] [Q&A]
Can docuseal be deployed to Azure container apps?  I don't think I'm specifying the environment variables correctly.

**Comment by omohokcoj** [comment]
@rfenyves yes docuseal can be deployed on Azure container apps - you can find the list of required and optional ENV vars here:
https://www.docuseal.co/docs/configuring-docuseal-via-environment-variables 

https://www.docuseal.co/docs/deploying-docuseal-on-azure-web-apps
